libs/ledger-live-common/src/families/ethereum/synchronisation.test.ts

```diff
@@ -48,3 +54,74 @@ describe("fetchAllTransactions", () => {
     expect(txs.length).toEqual(3);
   });
 });
+
+describe("preserveInitialOperationsDate", () => {
+  it("should preserve initial operations date in newOps", () => {
```

Comment:
@hzheng-ledger Don't we need to cover several different scenarios? (different date orders, etc...)

Reply:
fixed